packages/core/src/app/order/isOrderFee.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { getOrderFee } from "./orders.mock";
```

Comment:
Minor change, can you please rename this file `isOrderFee.test.ts`, we are trying to move away from spec naming.